tests/api_connexion/endpoints/test_dag_run_endpoint.py

```diff
@@ -1154,3 +1156,111 @@ def test_should_raises_403_unauthorized(self):
             environ_overrides={'REMOTE_USER': "test_view_dags"},
         )
         assert response.status_code == 403
+
+
+class TestPostSetDagRunState(TestDagRunEndpoint):
+    @parameterized.expand([("failed",), ("success",)])
+    @freeze_time(TestDagRunEndpoint.default_time)
+    def test_should_respond_200(self, state):
+        dag_id = "TEST_DAG_ID"
+        dag_run_id = "TEST_DAG_RUN_ID_1"
+        test_time = timezone.parse(self.default_time)
+        with create_session() as session:
+            dag = DagModel(dag_id=dag_id)
+            dag_run = DagRun(
+                dag_id=dag_id,
+                run_id=dag_run_id,
+                state=DagRunState.RUNNING,
+                run_type=DagRunType.MANUAL,
+                execution_date=test_time,
+                start_date=test_time,
+                external_trigger=True,
+            )
+            session.add(dag)
+            session.add(dag_run)
```

Comment:
It’s probably easier to use the `dag_maker` fixture for this (but the current implementation is OK).